test/automated/reflection/actuator/actuator.rb

```diff
@@ -0,0 +1,45 @@
+require_relative '../../automated_init'
+
+context "Reflect" do
+  context "Reflection" do
+    context "Actuator" do
+      constant_name = Controls::Namespace::Random.get
+      subject = Controls::Subject.example
+      target = Controls::Target.example(constant_name:, subject:)
+
+      method_name = :some_method
+      RecordInvocation.(target, method_name)
+
+      reflection = Reflect.(subject, constant_name)
+      reflection.(method_name, subject, :some_arg)
+
+      invocation = target.invocation(method_name)
+
+      context "Target's #{method_name} Method" do
+        test! "Invoked" do
+          refute(invocation.nil?)
+        end
+
+        context "Arguments" do
+          arguments = invocation.arguments
+
+          context "First Positional" do
+            value = arguments[:some_parameter]
+
+            test "Value" do
+              assert(value == subject)
+            end
```

Comment:
Run the test and observe whether the test needs a title at all (currently it's `"Value"`)